docs/usb/wdi-usb-interface.md

```diff
@@ -33,13 +38,29 @@ https://gitlab.freedesktop.org/libevdev/libevdev/-/blob/master/include/linux/lin
  BTN_THUMBR
 ```
 
-## Drive actions
+## Drive Actions
 | Action | Keyboard Controls | Gamepad Controls |
 |--------|------|-----|
 | Drive forward | 'w' \| up arrow | Y axis (towards min val) |
 | Drive backward | 's' \| down arrow | Y axis (towards max val) |
 | Drive left | 'a' \| left arrow | X axis (towards min val) |
 | Drive right | 'd' \| right arrow | X axis (towards max val) |
-| Emergency stop | backspace | button_east?? would like this to be something that is easy to press under stress |
-| Enable drive | enter | start button |
-| 
\ No newline at end of file
+| Emergency stop | backspace | BTN_SOUTH \| BTN_THUMBL?? |
+| Enable device control | enter | BTN_START |
+| Override | caps lock | BTN_EAST | 
+
+
+## Chair Control Actions
+Seating controls and user menu control match drive controls.
```

Comment:
Should we define controls for moving specific actuators? e.g. the right trigger moves the footplate, left trigger elevation, right analog stick tilt etc.

Reply:
I think the current proposal without controls for moving specific actuators is a great starting point. It best mirrors what people are used to and can be added to as ideas/new use cases need it.